soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
@jgkamat I will be travelling this weekend so I will have plenty of time on the train to look it over again. It would be nice if you would double check that everything runs properly though because my computer is pretty slow so it's hard to tell if the process is hanging due to the changes I made or just lag.